fs-ansible/roles/lustre/tasks/format.yaml

```diff
@@ -56,6 +63,13 @@
          state: present
       loop: "{{ osts.keys() }}"
 
+    - name: Wait for OST partition to appear in /dev
+      wait_for:
+        path: "/dev/{{ item }}p2"
+        timeout: 30
+        sleep: 5
+      loop: "{{ osts.keys() }}"
```

Comment:
Thanks, this looks great now, I hadn't thought of adding this, but its well worth a try, seems like it should fix things (or at least detect when problems happen)

Reply:
Thanks, I suspect occasionally /dev needs a bit of time to update to reflect changes to the partitions, and this should give it some time to sort itself out.